CPAC/pipeline/schema.py

```diff
@@ -61,8 +61,8 @@
         'segmentation': {
             'erode_mask': bool,
             'extraction_resolution': Any(
-                int, float, All(str, Match(resolution_regex))
-            ),
+                int, float, str), #All(str, Match(resolution_regex))
+            #),
```

Comment:
We just need to add `Functional` as an option here, right?

So it could be
```Python
'extraction_resolution': Any(
  int, float, 'Functional', All(str, Match(resolution_regex))
),
```

Reply:
Great catch, I forgot to clean this up- thanks!

Reply:
Alright- updated.